ATM90E36.cpp

```diff
@@ -324,12 +361,14 @@ double ATM90E36::GetImportEnergy() {
   // unsigned short senergyB = CommEnergyIC(READ, SenergyB, 0xFFFF);
   // unsigned short senergyC = CommEnergyIC(READ, SenergyC, 0xFFFF);
 
-  return (double)ienergyT / 100 / 3200; //returns kWh
+  return (double)(((ienergyT) /3.2)*3600); //returns kWh

```

Comment:
How 3.2 @mayurharge , We want to send in the joules, right ? Even if we send it in Joules, It should have been ienergyT /320 * 3600. 
Can you please confirm.